spec/system/exercises_system_spec.rb

```diff
@@ -1,162 +0,0 @@
-# frozen_string_literal: true
```

Comment:
So I understand that the test is flaky.
Could we use other measures than removing it entirely? I'm thinking about retrying it or marking it as flaky.